packages/client/katello-host-tools/katello-host-tools.spec

```diff
@@ -8,14 +8,15 @@
 
 %global build_agent (0%{?suse_version} == 0) && ((0%{?fedora} > 28) || (0%{?rhel} > 0))
 %global legacy_agent (0%{?rhel} == 5) || (0%{?rhel} == 6)
+%global build_fact_plugin ((0%{?rhel} > 0 && 0%{?rhel} <= 7) || (0%{?fedora} > 0 && 0%{?fedora} <= 24))
```

Comment:
We don't support Fedora <= 24 anymore so should we leave that out?

Reply:
This sounds like a good idea, yeah.